src/store/modules/blocks/actions.js

```diff
@@ -16,20 +16,13 @@ async function getGenerationFromHeightWrapper (height, epochUrl) {
   let microBlocks = await Promise.all(
     microBlocksHashes.map(
       async (hash) => {
-        return client.api.getMicroBlockHeaderByHash(hash, { txEncoding: 'json' })
+        let micro_block = await client.api.getMicroBlockHeaderByHash(hash, { txEncoding: 'json' })
```

Comment:
I have tried this APIs some days ago, and they work OK without `{ txEncoding: 'json' }`, is it necessary?